tests/accounting/test_simplepnl.py

```diff
@@ -47,6 +47,16 @@ def test_record_and_resolve_decision():
     actual_pnl = pnl_tracker.pnl_data[0][-1]
     assert actual_pnl == expected_pnl, f"Expected PnL={expected_pnl}, got {actual_pnl}"
 
+#
+# def test_simple_decisions():
```

Comment:
This simple test fails which is a sign of a bug in the implementation. But "fixing" the condition on decision breaks everything else. Needs a rewrite I feel. But would be good to sync with backend.